be/src/olap/rowset/CMakeLists.txt

```diff
@@ -46,3 +46,5 @@ add_library(Rowset STATIC
     beta_rowset.cpp
     beta_rowset_reader.cpp
     beta_rowset_writer.cpp)
+
+target_compile_options(Rowset PUBLIC "-Wno-error=maybe-uninitialized")
```

Comment:
Some variables in rowset is not initialized, so that I has to add this option to compile successfully.  I will add another PR to init all variables and remove this option.